docs/nodejs/getting-started.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started
+
+Getting started with Node.js on Altis Cloud is easy, no matter whether you're migrating an existing Next.js project or starting from scratch.
+
+
+## Before you start
+
+Before you get started, make sure you're aware of the requirements for Node.js on Altis Cloud.
+
+**Your subscription**: Node.js is an add-on to your existing Altis environments, and is not included as a standard part of your subscription. Contact your account manager for more information about adding Node.js to your subscription.
+
+**Next.js only**: Node.js on Altis Cloud only officially supports Next.js projects. See our [framework support page](./frameworks.md) for more details.
+
+**Attached to your application**: Node.js containers are attached to your existing Altis Cloud environment, so you'll need to [set up your WordPress-based application first](docs://getting-started/).
+
+
+## Starting your project
+
+### Starting a new project
+
+We recommend starting with the [Altis Node.js Skeleton project](https://github.com/humanmade/altis-nodejs-skeleton). This project includes a sample Next.js project and build script, ready to be deployed to your environment.
+
+To get started, clone this repository to a new repository you own.
+
+
+### Adapting your existing project
+
+If you already have a Next.js-based project, you can easily adapt it to run on Altis Cloud.
+
+You'll need to make sure you meet the following requirements:
+
+* Altis runs `npm start` to start your application; ensure your `package.json` runs the standard `next start` command for `scripts.start`
```

Comment:
In the other documents it says "npm run start" I think they work out the same, but we should be consistent.